changelog/2022-02-25T10_53_17+01_00_evaluator_float_double.md

```diff
@@ -0,0 +1,2 @@
+FIXED: The evaluator rule for unpack{Float,Double}# are now corrected to return boxed float and double instead of unboxed literals.
```

Comment:
```suggestion
FIXED: The evaluator rule for unpack{Float,Double}# are now corrected to return boxed float and double instead of unboxed literals. [#2097](https://github.com/clash-lang/clash-compiler/issues/2097)
```

Reply:
How could I forget the links :see_no_evil: